### PR TITLE
Feature/api/trs

### PR DESCRIPTION
**Pull request by askarovit**

### CODE REVIEW

File: api/repository/transaction.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { generateTrs } from '../mock/transactions';
+import { IListContainer } from '../util/common';
+import Response from 'shared/model/response';
+import { IAsset, Transaction } from 'shared/model/transaction';
+
+class TransactionRepository {
```
Comment (macaqueee) [inline api/repository/transaction.ts]: place this repository in shared

File: api/controller/block/index.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import Response from 'shared/model/response';
+import { reqGetBlocks } from 'api/controller/block/types';
+import { RPC } from 'api/utils/decorators';
+
+class BlockController {
+
+    @RPC('GET_BLOCKS_HISTORY')
+    public getBlocksHistory(data?: reqGetBlocks) {
+        const blocks: Response<Array<Block>> = BlockService.getMany(data);
+        return blocks.data;
```
Comment (macaqueee) [inline api/controller/block/index.ts]: return `ResponseEntity` here. Not only pure data

Comment (macaqueee) [inline api/controller/block/index.ts]: return `ResponseEntity` here. Not only pure data.

Comment (askarovit) [reply api/controller/block/index.ts]: I agree

**Pull request closed without merging**